target/linux/ath79/dts/qca9531_telco_t1.dts

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "telco,t1", "qca,qca9531";
+	model = "Telco T1";
+
+	aliases {
+		label-mac-device = &eth0;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &pinmux_led_wan_pin>;
+
+		wan {
+			label = "t1:blue:wan";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		lan {
+			label = "t1:blue:lan";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "t1:blue:wlan";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		rssi0 {
+			label = "t1:blue:rssi0";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi1 {
+			label = "t1:blue:rssi1";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		rssi2 {
+			label = "t1:blue:rssi2";
```

Comment:
Are these really blue? On the product page, I read something about red/yellow/green?
Apart from that, one could use one of these for boot indication via aliases, i.e. as it's done for TP-Link CPEs: https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/ar9344_tplink_cpe_2port.dtsi#L10

Reply:
Yes, the LEDs are blue.  Forgot to add indication LEDs, thanks for the reminder